rosa_architecture/rosa-understanding.adoc

```diff
@@ -5,7 +5,59 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: rosa-understanding
 toc::[]
 
-Learn about {product-title} (ROSA) access, supported consoles, consumption experience, and integration with Amazon Web Services (AWS) services.
+Learn about {product-title} (ROSA), interacting with ROSA using {cluster-manager-first} and command-line interface (CLI) tools, consumption experience, and integration with Amazon Web Services (AWS) services.
 
-include::modules/rosa-understanding.adoc[leveloffset=+1]
-include::modules/rosa-using-sts.adoc[leveloffset=+2]
+== About ROSA
+
+ROSA is a fully-managed turnkey application platform that allows you to focus on delivering value to your customers by building and deploying applications. Red Hat and AWS Site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
+
+You subscribe to the service directly from your AWS account.
+
+After the clusters are created, you can operate your clusters with the OpenShift web console or through {cluster-manager-first}. The ROSA service also uses OpenShift APIs and command-line interface (CLI) tools. These tools provide a standardized OpenShift experience to use your existing skills and tools knowledge.
+
+You receive OpenShift updates with new feature releases and a shared, common source for alignment with OpenShift Container Platform. ROSA supports the same versions of OpenShift as Red Hat OpenShift Dedicated and OpenShift Container Platform to achieve version consistency.
+
+== Credential modes
+
+There are two supported credential modes for ROSA clusters. One uses the AWS Secure Token Service (STS), which is recommended, and the other uses Identity Access Management (IAM) roles.
+
+=== AWS STS
```

Comment:
This should be "ROSA with STS".  Also we should put in parenthesis that this is the recommended mode with "(recommended)".  So together the header would be "ROSA with STS (recommended)".

Reply:
We restricted by style guidelines on the parentheses, but it's included in the text.